wp-content/themes/engage/templates/partial/teammate.twig

```diff
@@ -5,15 +5,16 @@
         </div>
     {% endif %}
     <div class="mate__content">
-      {% if mate.post_type == "team" %}
-        <a class="mate__name" href="/team/{{mate.slug}}">{{ mate.name }}</a>
+      <h3>
+          <a class="mate__name--{{ format }}" {{ format == "preview" ? "href=#{mate.link}" }}>{{ mate.name }}</a>
```

Comment:
This is _close_, but if there's no `href`, then it shouldn't be a link. If there's no link (full), it should look like:
`<h3>{{ mate.name}}</h3>`
and if there's a link (preview):
`<h3><a href="{{mate.link}}">{{ mate.name}}<</a></h3>`

Rule of thumb, if there's no link (or you'll sometimes see a fake one like `href="#"`) then it should _not_ be a link (`<a>`) but something else.